src/jsDelivr.js

```diff
@@ -0,0 +1,35 @@
+import got from 'got';
+import c from './config.js';
+import log from './log.js';
+
+const hits = new Map();
+
+function formatHits(pkg) {
+  if (pkg.type !== 'npm') {
+    return;
+  }
+
+  hits.set(pkg.name, pkg.hits);
+}
+
+export async function loadHits() {
+  const hitsJSONpromise = got(c.jsDelivrHitsEndpoint, { json: true }).catch(
+    error => {
```

Comment:
I was wondering if it would be better to reject with the error here because if this fails at the start there won't be hits for any packages.

Reply:
yes, we should probably reject here, since otherwise every package will log if this can't be loaded